website/.spelling

```diff
@@ -324,6 +324,7 @@ deserializes
 downtimes
 druid
 druid–kubernetes-extensions
+druid.table
```

Comment:
Did it fail the spell check? If not, I don't think you'd need to add it since the spell check shouldn't run inside code blocks.

Reply:
@317brian was having issues with it failing the spell check and suggested I had it here